### PR TITLE
Iterate card component

### PR DESCRIPTION
**Pull request by Sushmareddy-OSI**
Card component is updated by adding props for grid.

### CODE REVIEW

**Comment by edgibbs** [comment]
Can one of the admins verify this patch?

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
Build triggered. sha1 is merged.

**Comment by edgibbs** [comment]
Build started sha1 is merged.

**Comment by edgibbs** [comment]
Everything is green!
 No test results found.

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
Build triggered. sha1 is merged.

**Comment by edgibbs** [comment]
Build started sha1 is merged.

**Comment by edgibbs** [comment]
Everything is green!
 No test results found.

File: src/Cards.js [hunk 1]
```diff
@@ -1,23 +1,66 @@
...
-const Cards = props => (
-  <div className="bg-lightest-grey container ">
-    <div className="card edit double-gap-top">
-      <div className="card-header">
-        <span>{props.cardHeaderText}</span>
-      </div>
-      <div className="card-body">
-        {props.children}
-        <div className="clearfix" />
+const Cards = props => {
```
Comment (edgibbs) [inline src/Cards.js]: So totally don't have to change for this pull request, but don't know why we call the component Cards.  I'll have Alyssa do a quick check.

Comment (NagaPradeepGajula-osi) [reply src/Cards.js]: Do you mean that the name should be "Card" instead of "Cards"?

File: src/Cards.js [hunk 2]
```diff
@@ -1,23 +1,66 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import classNames from 'classnames';
```
Comment (eckdanny-osi) [inline src/Cards.js]: Plz add `classnames` to dependencies if importing the module directly.

Comment (NagaPradeepGajula-osi) [reply src/Cards.js]: ^ done.

File: src/Cards.js [hunk 3]
```diff
@@ -1,23 +1,66 @@
...
+  const classField = classNames(
+    cardbgcolor,
+    wrapContainer,
+    `col-lg-${columnLargeWidth}`,
+    `col-md-${columnMediumWidth}`,
+    `col-md-offset-${offsetMediumValue}`,
+    `col-xs-${columnXsmallWidth}`
+  );
+  let editClass = '';
+  if (props.editable) {
```
[the remark this answers is not in the document]
Comment (NagaPradeepGajula-osi) [reply src/Cards.js]: we have updated this yesterday with following commit 051666f.

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
@alyssaburkecwds approved card design for this iteration.

**Comment by edgibbs** [comment]
Build triggered. sha1 is merged.

**Comment by edgibbs** [comment]
Build started sha1 is merged.

**Comment by edgibbs** [comment]
Everything is green!
 No test results found.

**Comment by NagaPradeepGajula-osi** [comment]
@edgibbs can we merge this? We can use the changes of card in Case-Management client profile.

**Pull request merged**